packages/elements/src/clock/index.ts

```diff
@@ -130,6 +132,12 @@ export class Clock extends ResponsiveElement {
   @state()
   private tickTimestamp = 0;
 
+  /**
+   * Current active state for interactive
+   */
+  @state()
+  private active = 'hours';
```

Comment:
1. Create `enum` to describe possible state. `active` is too common name.
2. Rename it to e.g. `activeSegment`